src/docs/controls/CheckBox.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: control
+name: CheckBox
+group: controls
+---
+[CheckBox]: https://avaloniaui.net/docs/controls/checkbox
+[CheckBox API]: https://avaloniaui.net/api/Avalonia.Controls/CheckBox/
+[CheckBox.fs]: https://github.com/AvaloniaCommunity/Avalonia.FuncUI/blob/master/src/Avalonia.FuncUI.DSL/CheckBox.fs
+
+> *Note*: You can check the Avalonia docs for the [CheckBox] and [CheckBox API] if you need more information.
+> 
+> For Avalonia.FuncUI's DSL properties you can check [CheckBox.fs]
+
+The checkbox is a control that allows a user to represent boolean values or the absense of a value
+
+## Usage
+
+**Set Label**
+```fsharp
+CheckBox.create [
+    Checkbox.content "I Accept the terms and conditions."
+]
+```
+
+**Set Is Checked**
+
+```fsharp
+CheckBox.create [
+    // can be either true or false
+    Checkbox.isChecked state.booleanValue
+]
+```
+
+**Set Indeterminate**
+```fsharp
+CheckBox.create [
+    // can be either true or false
```

Comment:
`true` / `false` or `null` right ?

Reply:
I played a little bit with this yesterday it seems you need to have both `isThreeState` in true and `isChecked` in null (None or nullable bool set to null) for it to work  or else it will show as unselected (`isChecked` false) it's a bit weird actually